modules/common/src/main/scala/surge/internal/kafka/HostAssignmentTracker.scala

```diff
@@ -52,8 +52,13 @@ object HostAssignmentTracker {
   private case object ClearState
 
   private case class ClusterState(state: Map[TopicPartition, HostPort])
-  private class HostAssignmentTrackerImpl extends Actor {
-    override def receive: Receive = receiveWithState(ClusterState(Map.empty))
+  private class HostAssignmentTrackerImpl extends Actor with Stash {
+    override def receive: Receive = uninitialized()
+
+    private def uninitialized(): Receive = {
+      case msg: UpdateState => handleUpdateState(ClusterState(Map.empty), msg)
```

Comment:
Should we just `unstashAll()` here rather than in `handleUpdateState` since we're only ever stashing messages from the `uninitialized` state?